### ISSUE TITLE
traefik 1.7: Error creating server

### ISSUE THREAD

**Issue by r0bj** [containous/traefik#3949]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Setting TLS for entrypoint `traefik` failed with traefik 1.7 with error message:
```
time="2018-09-28T16:42:19Z" level=error msg="Error creating server: open : no such file or directory"
```

Traefik 1.6.6 works fine with the same configuration.

Deployment:
```yml
apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    k8s-app: traefik-ingress-controller
  name: traefik-ingress-controller
  namespace: kube-system
spec:
  selector:
    matchLabels:
      k8s-app: traefik-ingress-controller
  replicas: 1
  template:
    metadata:
      labels:
        k8s-app: traefik-ingress-controller
    spec:
      serviceAccountName: traefik-ingress-controller
      containers:
      - name: traefik-ingress-controller
        image: traefik:v1.7.0-alpine
        args:
        - --kubernetes
        - --api
        - --entrypoints=Name:http Address::80 Redirect.EntryPoint:https
        - --entrypoints=Name:https Address::443 TLS
        - --entrypoints=Name:traefik Address::8080 TLS
        - --defaultentrypoints=http,https
        - --traefiklog
        - --acme.email=redacted@example.com
        - --acme.storage=/traefik/acme/account
        - --acme.httpchallenge.entrypoint=http
        - --acme.entrypoint=https
        - --acme.onhostrule
        - --acme.caserver=https://acme-staging-v02.api.letsencrypt.org/directory
        - --etcd.endpoint=etcd-traefik-client:2379
        - --etcd.useapiv3
        ports:
        - containerPort: 80
          name: http
        - containerPort: 443
          name: https
        - containerPort: 8080
          name: https-gui
```

### What did you expect to see?

Traefik 1.7 is behaving similar to version 1.6 and serving api on port 8080 with TLS:

```
curl -svko /dev/null https://example.com:8080/dashboard/
[...]
< HTTP/1.1 200 OK
```

### What did you see instead?

Traefik 1.7 generates error message during start:
```
time="2018-09-28T16:42:19Z" level=error msg="Error creating server: open : no such file or directory"
```

Traefik 1.7 is not serving any traffic on port 8080:
```
curl -svko /dev/null https://example.com:8080/dashboard/
[...]
timeout
```

**Comment by ldez** [comment]
@r0bj could you provide more logs?

**Comment by r0bj** [comment]
@ldez sure:
https://gist.github.com/r0bj/ce3b5b25ecb024e2e39b670b2c09aae8

But there is just one related line I found:
```
time="2018-09-28T18:56:38Z" level=error msg="Error creating server: open : no such file or directory"
```

**Comment by ldez** [comment]
@r0bj we found the bug and we fixed it.

For now, as workaround, your need to remove `TLS` from your `traefik` (8080) entry point.

**Comment by traefiker** [comment]
Closed by #3962.

**Comment by r0bj** [comment]
Thanks @ldez
